Gemfile.lock

```diff
@@ -364,11 +363,12 @@ GEM
       rainbow (>= 2.2.2, < 4.0)
       ruby-progressbar (~> 1.7)
       unicode-display_width (>= 1.4.0, < 1.7)
-    rubocop-rails (2.4.0)
+    rubocop-rails (2.5.0)
+      activesupport
       rack (>= 1.1)
       rubocop (>= 0.72.0)
     ruby-progressbar (1.10.1)
-    ruby-vips (2.0.16)
+    ruby-vips (2.0.17)
```

Comment:
### [ruby-vips](http://github.com/libvips/ruby-vips)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 2.0.16 → 2.0.17

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **49** commits. See the full changes on [the compare page](https://github.com/libvips/ruby-vips/compare/v2.0.16...v2.0.17).

These are the first 10 commits:
- (2a4ad2e) [Add libvips dependency to gemspec which is respected by Ruby…](https://github.com/libvips/ruby-vips/commit/2a4ad2ec542ed4ad400d0da2059f318897b0ec21)
- (043e9b0) [fix rubocop ... again](https://github.com/libvips/ruby-vips/commit/043e9b0f385a37d8428cd57214d88201dc1a57bb)
- (6ad8026) [Merge pull request #211 from larskanis/add-mingw-dep](https://github.com/libvips/ruby-vips/commit/6ad8026e07658c9d15d667c1670a7712be463e6a)
- (24a0621) [version bump to 2.0.17](https://github.com/libvips/ruby-vips/commit/24a06216038cf643b6867bed6a3cb29d85dadd8f)
- (69d871c) [disable rubocop durinig CI](https://github.com/libvips/ruby-vips/commit/69d871ce08481150c813cd194dd3062c3f9f61ec)
- (cc85f03) [Merge branch 'master' of github.com:libvips/ruby-vips](https://github.com/libvips/ruby-vips/commit/cc85f03d2252f318aa1fbc8ad7a31cfcd64324c1)
- (5c67338) [tone down the blurb](https://github.com/libvips/ruby-vips/commit/5c673386e131a817360bd43c9896b6dd4e281efe)
- (665f78b) [fix typo in comments](https://github.com/libvips/ruby-vips/commit/665f78bd2456613593b4e1bfaf3fe5ad3f7502c1)
- (ba16b13) [better - to _ conversion](https://github.com/libvips/ruby-vips/commit/ba16b13e6f01795303dd13a4045718d8d656c11c)
- (28b0253) [Merge branch 'master' of github.com:libvips/ruby-vips](https://github.com/libvips/ruby-vips/commit/28b0253d863b13377b19b97cdf8fc69728e1fe8b)


</details>